lib/js/helper.go

```diff
@@ -117,7 +117,7 @@ var WaitIdle = &Function{
 // WaitLoad ...
 var WaitLoad = &Function{
 	Name:         "waitLoad",
-	Definition:   `function(){const n=this===window;return new Promise((e,t)=>{if(n){if("complete"===document.readyState)return e();window.addEventListener("load",e)}else void 0===this.complete||this.complete?e():(this.addEventListener("load",e),this.addEventListener("error",t))})}`,
+	Definition:   `function(){const n=this===window;return new Promise((e,t)=>{if(n){if("complete"===document.readyState)return e();window.addEventListener("load",_=>e())}else void 0===this.complete||this.complete?e():(this.addEventListener("load",_=>e()),this.addEventListener("error",t))})}`,
```

Comment:
Nice catch! If you check the first line of `helper.go`, it's actually generated by `helper.js`:

https://github.com/go-rod/rod/blob/eff7c9785ce2ae78231d5fdce51c32a4e76af2b4/lib/js/helper.js#L1

Could you modify the helper.js then run `go generate`?

Reply:
done! thanks!